### PR TITLE
Add CLI tool

### PR DESCRIPTION
**Pull request by ProZsolt**

### CODE REVIEW

File: bin/midea-ac [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    def print_table(values, head)
+      head.each do |title|
+        title[:lenght] = (values.map{|value| value[title[:id]].to_s.length} + [title[:name].length]).max
+      end
+      print_line(head)
+      print_head(head)
+      print_line(head)
+      values.each{|row| print_row(head, row)}
+      print_line(head)
+
```
Comment (yitsushi) [inline bin/midea-ac]: Trailing whitespace

File: bin/midea-ac [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    end
+
+    def print_table(values, head)
+      head.each do |title|
+        title[:lenght] = (values.map{|value| value[title[:id]].to_s.length} + [title[:name].length]).max
+      end
+      print_line(head)
+      print_head(head)
+      print_line(head)
+      values.each{|row| print_row(head, row)}
```
Comment (yitsushi) [inline bin/midea-ac]: Space between { and | missing.

File: bin/midea-ac [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+          builder.finalize
+        )
+      rescue RuntimeError => e
+        if e.message == 'the appliance is off line.'
+          raise Thor::Error, 'The device is offline.'
+        else 
+          raise e
+        end
+      end
+
```
Comment (yitsushi) [inline bin/midea-ac]: Trailing whitespace

File: bin/midea-ac [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    puts "TimerOff is #{(device.off_timer[:status] ? '' : 'not')} active."
+    puts "  at: #{device.off_timer_human}" if device.off_timer[:status]
+    puts "Eco mode is #{(device.eco ? 'on' : 'off')}."
+  end
+
+  desc 'set ID', 'Sets device\'s power, temperature and fan speed'
+  method_option :power, required: false, type: :string, aliases: "-p", desc: "Power"
+  method_option :target_temperature, required: false, type: :numeric, aliases: "-t", desc: "Target temperature"
+  method_option :fan_speed, required: false, type: :numeric, aliases: "-f", desc: "Fan speed"
+  def set id
```
Comment (yitsushi) [inline bin/midea-ac]: Use def with parentheses when there are parameters.

File: bin/midea-ac [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+    puts "TimerOn is #{(device.on_timer[:status] ? '' : 'not')} active."
+    puts "  at:  #{device.on_timer_human}" if device.on_timer[:status]
+    puts "TimerOff is #{(device.off_timer[:status] ? '' : 'not')} active."
+    puts "  at: #{device.off_timer_human}" if device.off_timer[:status]
+    puts "Eco mode is #{(device.eco ? 'on' : 'off')}."
+  end
+
+  desc 'set ID', 'Sets device\'s power, temperature and fan speed'
+  method_option :power, required: false, type: :string, aliases: "-p", desc: "Power"
+  method_option :target_temperature, required: false, type: :numeric, aliases: "-t", desc: "Target temperature"
```
Comment (yitsushi) [inline bin/midea-ac]: Prefer single-quoted strings when you don't need string interpolation or special

**Comment by ProZsolt** [comment]
Updated!

**Pull request merged**